### PR TITLE
4.10.0 NEWS

### PR DESCRIPTION
**Pull request by adamdickmeiss**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -15,7 +34,7 @@ Fixes:
  * [OKAPI-1031](https://issues.folio.org/browse/OKAPI-1031) "apt-get install okapi" should recreate `/var/lib/okapi`
 
 Other:
- * [OKAPI-1035](https://issues.folio.org/browse/OKAPI-1035) Upgrade to Vert.x 4.1.4 OKAPI-1035
+ * [OKAPI-1035](https://issues.folio.org/browse/OKAPI-1035) Upgrade to Vert.x 4.1.4 [OKAPI-1035](https://issues.folio.org/browse/OKAPI-1035)
```
Comment (julianladisch) [inline NEWS.md:37]: `OKAPI-1035` is already at the beginning of the line, can be removed from the end of line

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,22 @@
+## 4.10.0 2021-12-13
+
+Fixes:
+ * [OKAPI-1050](https://issues.folio.org/browse/OKAPI-1050) -Dlog4j2.formatMsgNoLookups=true for Debian/Ubuntu package (CVE-2021-44228)
+ * [OKAPI-1049](https://issues.folio.org/browse/OKAPI-1049) Disable JDNI by removing JdniLookup class
```
Comment (julianladisch) [inline NEWS.md:5]: OKAPI-1047, not OKAPI-1049

Comment (adamdickmeiss) [reply NEWS.md]: ops. thx

**Review by julianladisch — APPROVED** [review]
Looks good, thanks!

**Pull request merged**